view/src/components/foodbanks.js

```diff
@@ -0,0 +1,926 @@
+                          <Chip label={option} {...getTagProps({ index })} />
+                        ))
+                      }
+                      renderInput={(params) => (
+                        <TextField
+                          {...params}
+                          variant="outlined"
+                          label="Food Bank Tags"
+                          placeholder="tags..."
+                        />
+                      )}
+                    />
+                  </Grid>
+                </Grid>
+              </form>
+            </Container>
+          </Dialog>
+          <Container maxWidth="lg">
+            <Grid container spacing={2} alignItem="center">
+              <Grid item xs={12}>
+                <div className={classes.search}>
+                  <div className={classes.searchIcon}>
+                    <SearchIcon />
+                  </div>
+                  <InputBase
+                    fullWidth={true}
+                    placeholder="Search…"
+                    classes={{
+                      root: classes.inputRoot,
+                      input: classes.inputInput,
+                    }}
+                    inputProps={{ "aria-label": "search" }}
+                  />
+                </div>
+              </Grid>
+              {this.state.foodbanks.map((foodbank) => (
+                <Grid item xs={12}>
+                  <Card className={classes.root} variant="outlined">
+                    <CardContent>
+                      <Typography variant="h5" component="h2">
+                        {foodbank.foodbankName}
+                      </Typography>
+                      <Chip
+                        className={classes.chip}
+                        label="High Food Insecurity"
+                        size="small"
+                      />
+                      <Chip
+                        className={classes.chip}
+                        label="Major City"
+                        size="small"
+                      />
+                      <Box
+                        display="flex"
+                        flexDirection="row"
+                        flexWrap="wrap"
+                        p={0}
+                        m={0}
+                      >
+                        <Box p={3}>
+                          <Typography
+                            className={classes.pos}
+                            color="textSecondary"
+                          >
+                            Details:
+                          </Typography>
+                          <Typography
+                            variant="body2"
+                            component="p"
+                            className={classes.foodbankLocation}
+                          >
+                            Location of Food Bank: {foodbank.location}
+                            <br />
+                            Refridgeration Space (in pallets):
```

Comment:
Refrigeration (and yet "fridge" does have a "d" in it...  told you programming gets philosophical a lot).